policyengine_us/variables/gov/local/co/denver/dhs/property_tax_relief/eligibility/co_de_property_tax_relief_homeowner_eligible.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class co_de_property_tax_relief_homeowner_eligible(Variable):
+    value_type = float
+    entity = SPMUnit
+    unit = USD
+    label = "Eligible for the homeowner Denver Property Tax Relief"
+    definition_period = YEAR
+    reference = "https://denvergov.org/files/content/public/v/37/government/agencies-departments-offices/agencies-departments-offices-directory/denver-human-services/be-supported/additional-assistance/property-tax-relief/denver-property-tax-relief-program-year-2021-rules.pdf"
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(period).gov.local.co.denver.dhs.property_tax_relief
+        has_property_taxes = add(spm_unit, period, ["real_estate_taxes"]) > 0
+        has_elderly_or_disabled = spm_unit("has_usda_elderly_disabled", period)
```

Comment:
Denver ties its elderly/disabled definition to the USDA's?

Scanning it looks like it's based on age 65. USDA is 60. Should be a policy parameter if they define it themselves.